lisp/forge-commands.el

```diff
@@ -893,6 +895,12 @@ is added anyway.  Currently this only supports Github and Gitlab."
 
 ;;; Misc
 
+(transient-define-infix forge-forge.max-topics ()
+  "Change the maximum number of topics to pull at once."
+  :class 'magit--git-variable
+  :variable "forge.max-topics"
```

Comment:
We usually do what Git does and use camelCase. `forge.graphqlItemLimit` and `forge-forge.graphqlItemLimit` sound about right.

Also, since this is less important than other infixes, place it this after the definitions of those, probably right before the database utilities at the end of this section.

Reply:
Sure, I wasn't sure which convention to prefer and I'm sure I found a dashed one; not a problem to change :)